### ISSUE TITLE
Information panels - WCAG 2.1 2. Keyboard Trap 

### ISSUE THREAD

**Issue by mathalete** [bbc/gef-docs#183]
On our Information panel tech doc page it suggests 

> In order to achieve modality, dialog implementations often 'trap' focus within them. Even in a dialog, this approach to focusing the user's attention is not recommended, since it makes it difficult to leave the page. In fact, it is a failure under WCAG2.1 2.1.2 Keyboard Trap[[6]](https://bbc.github.io/gel/components/info-panels/#fn6) and use of inert is preferred (see the [Action dialog](https://bbc.github.io/gel/components/action-dialogs)).

It's not actually a failure of this WCAG SC.

Recommend revisiting this and reframing in terms of MAG.
